src/psyclone/psyir/nodes/omp_directives.py

```diff
@@ -107,13 +111,32 @@ def _get_reductions_list(self, reduction_type):
                             result.append(arg.name)
         return result
 
+    @property
+    def clauses(self):
+        '''
+        :returns: the Clauses associated with this directive.
+        :rtype: List of :py:class:`psyclone.psyir.nodes.Clause`
+        '''
+        if len(self.children) > 1:
+            return self.children[1:]
+        return []
+
```

Comment:
I resolved the previous comment because I couldn't reply but I still think this should be in RegionDirective. The behaviour will be the same in OpenACC even if we don't have examples of it yet.

Same for the OMPStandaloneDirective.clauses I think it could go to the generic StandaloneDirective

Reply:
Moved these up to the parents then